spec/game_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'game' 
+require 'board'
+
+describe Game do
+  EMPTY_BOARD = Board.empty_board
+
+  context 'new game' do
+    it 'has an empty board' do
```

Comment:
This test doesn't serve any purpose; there's no need for Game to expose board here